dev/com.ibm.ws.security.openidconnect.server/resources/com/ibm/ws/security/openidconnect/server/internal/resources/OidcServerMessages.nlsprops

```diff
@@ -180,6 +180,9 @@ OIDC_SERVER_JWK_NOT_AVAILABLE=CWWKS1640W: A request for a JWK JSON string could
 OIDC_SERVER_JWK_NOT_AVAILABLE.explanation=Producing a JWK JSON string requires the OpenID Connect provider signatureAlgorithm attribute to be set to RS256. Attributes KeyStoreRef and keyAliasName must be valid if specified. 
 OIDC_SERVER_JWK_NOT_AVAILABLE.useraction=Correct the OpenID Connect provider configuration attributes.
 
+OIDC_SERVER_THIRDPARTY_IDTOKEN_ERROR=CWWKS1641E: The claims from the third-party ID token could not be propagated for OpenID Connect provider {0}. 
+OIDC_SERVER_THIRDPARTY_IDTOKEN_ERROR.explanation=There was an issue with extracting the claims from the third-party ID token.
```

Comment:
rewrite as
The claims from the third-party ID token cannot be extracted.

Reply:
I'd also add in a statement like "The error message contains more information about why the claims cannot be extracted."

Reply:
i've included both your suggestions, ptal